modules/oe_theme_content_publication/src/Plugin/ExtraField/Display/PublicationDates.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_content_publication\Plugin\ExtraField\Display;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Datetime\DateFormatterInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\extra_field\Plugin\ExtraFieldDisplayFormattedBase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Display EU contribution and its percentage of the total budget.
+ *
+ * @ExtraFieldDisplay(
+ *   id = "oe_theme_content_publication_dates",
```

Comment:
I think the id is a bit confusing since the ct is called Publication and we have a date format "oe_theme_content_publication_date". We could try:
```suggestion
 *   id = "oe_publication_date",
```

Reply:
Date format and extra field are different entities. Best practice is to use module name in the plugin id. In this project plugins have different naming - with and without module name in id.